AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -66,6 +67,9 @@ - (instancetype)init {
                                selector:@selector(applicationWillEnterForeground)
                                    name:UIApplicationWillEnterForegroundNotification
                                  object:nil];
+
+    // Migrate data from previous versions.
+    [MSKeychainUtil migrateDistributeData];
```

Comment:
Should we do this before accessing any of user defaults? I think this should be in line 52.

Reply:
We can do it earlier yes as it's just model.